tests/w2w_test.py

```diff
@@ -103,13 +161,50 @@ def test_create_wrf_gridinfo(tmpdir):
     }
     create_wrf_gridinfo(info)
     assert os.listdir(tmpdir) == ['dst_gridinfo.tif']
-    tif = rioxarray.open_rasterio(info['dst_gridinfo'])
+    tif = rxr.open_rasterio(info['dst_gridinfo'])
     assert tif.rio.crs.to_proj4() == '+init=epsg:4326'
     assert tif.rio.transform() == Affine(
             0.01000213623046875, 0.0, -1.4050254821777344,
             0.0, 0.010000228881835938, 41.480000495910645,
     )
 
+def test_get_SW_BW(capsys):
+
+    ucp_table = pd.read_csv('testing/LCZ_UCP_lookup.csv',
```

Comment:
We can simply read the same file, so we don't have to have the exact same file twice in the repo so: `pd.read_csv('w2w/resources/LCZ_UCP_lookup.csv', index_col=0)` should be fine. `sep=','` is the default btw.

Reply:
see #41 -- I added a test for both cases